ocs_ci/ocs/elasticsearch.py

```diff
@@ -251,14 +265,21 @@ def _deploy_es(self):
 
         # Creating PVC for the elasticsearch server and wait until it bound
         log.info("Creating 10 GiB PVC for the ElasticSearch cluster on")
-        self.pvc_obj = create_pvc(
-            sc_name=constants.CEPHBLOCKPOOL_SC,
-            namespace=self.namespace,
-            pvc_name="elasticsearch-data-quickstart-es-default-0",
-            access_mode=constants.ACCESS_MODE_RWO,
-            size="10Gi",
-        )
-        wait_for_resource_state(self.pvc_obj, constants.STATUS_BOUND)
+        try:
+            self.pvc_obj = create_pvc(
+                sc_name=self.args.get("sc") or constants.CEPHBLOCKPOOL_SC,
+                namespace=self.namespace,
+                pvc_name="elasticsearch-data-quickstart-es-default-0",
+                access_mode=constants.ACCESS_MODE_RWO,
+                size="10Gi",
+            )
+
+            # Make sure the PVC bound, or delete it and raise exception
+            wait_for_resource_state(self.pvc_obj, constants.STATUS_BOUND)
```

Comment:
Do we have the validation if this PVC exists or not, sorry if I missed here in the coding somewhere. also would suggest to add in cleanup function too.

Reply:
before creation of the PVC, there is a validation that the namespace is not exist (or delete it if needed) at lines 134-143, so this will make sure the PVC is not exist at this point.